homeassistant/components/flux_led/light.py

```diff
@@ -96,6 +103,25 @@
     ),
 }
 
+SET_MUSIC_MODE_DICT: Final = {
+    vol.Optional(ATTR_SENSITIVITY, default=100): vol.All(
+        vol.Range(min=0, max=100), vol.Coerce(int)
```

Comment:
Maybe change the order of validators and coerce to integer first before validating the range?

Reply:
Will do tomorrow or if the wifi works in flight. 👍

Reply:
https://github.com/home-assistant/core/pull/62509